cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -2,7 +2,8 @@
 
 set -e
 
-_cli_container="kubevirtci/gocli@sha256:caf1c3f63dc5f3137795084e492a48c802a7ef2e7f7fbe1d67e5cff684d376a3"
+#_cli_container="kubevirtci/gocli@sha256:caf1c3f63dc5f3137795084e492a48c802a7ef2e7f7fbe1d67e5cff684d376a3"
+_cli_container="bazel:gocli"
```

Comment:
You can push a new gocli image with `/test push-gocli`. After that you need to check logs of that job and get the new shasum from them, and put it here and in the readme

Reply:
please switch this back to the remote gocli image :slightly_smiling_face: